generators/entity/templates/server/src/main/java/package/common/field_validators.ejs

```diff
@@ -26,44 +26,44 @@ if (field.fieldValidate === true) {
     const validators = [];
     const MAX_VALUE = 2147483647;
 
-    if (rules.indexOf('required') !== -1) {
+    if (rules.includes('required')) {
         validators.push('@NotNull');
     }
-    if (rules.indexOf('minlength') !== -1 && rules.indexOf('maxlength') === -1) {
+    if (rules.includes('minlength') && rules.includes('maxlength')) {
```

Comment:
`rules.indexOf('minlength') !== -1 && rules.indexOf('maxlength') === -1` should be `rules.includes('minlength') && !rules.includes('maxlength')`
Note the `=== -1`

Reply:
done